### PR TITLE
Device Samples To .NET Core 3.1

### PR DESCRIPTION
**Pull request by djaus2**
Whilst most of the projects remain set to .NET Core 2.1 for backward compatibility, the sample projects are set to .NET Core 3.1. This applies to:
- The sample projects with each device as well as 
- The /samples example projects

. .NET Standard projects remain at version 2.0.

REAME.md has been updated to include this message.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=1143) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=1143) before we can accept your contribution.<br/><br/>:x: djaus2 [sign now](https://cla.dotnetfoundation.org/dotnet/iot?pullRequest=1143)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/iot?pullRequest=1143) it.</sub>

**Comment by richlander** [comment]
I like the direction. We should continue to move the samples to the latest supported version of .NET.

@joperezr or @krwq are the ones to review the PR.

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="5.0.0-beta.20330.3">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="5.0.0-beta.20364.3">
```
Comment (joperezr) [inline eng/Version.Details.xml:4]: Why did you need to update these versions? I wouldn't expect you to need to update these.

File: eng/Versions.props [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
-    <MicrosoftDotNetGenAPIPackageVersion>5.0.0-beta.20330.3</MicrosoftDotNetGenAPIPackageVersion>
+    <MicrosoftDotNetGenAPIPackageVersion>5.0.0-beta.20364.3</MicrosoftDotNetGenAPIPackageVersion>
```
Comment (joperezr) [inline eng/Versions.props:5]: Same here, I wouldn't think you need to update this one as well as everything under eng\common

File: README.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ The repository also contains [Iot.Device.Bindings](https://www.nuget.org/package
 
 **NOTE**: This repository is still in experimental stage and all APIs are subject to changes.
 
+# .NET Core Versions
+
+Whilst most of the projects are set to .NET Core 2.1 for backward compatibility, the sample projects are set to .NET Core 3.1. This applies to the sample projects with each device as well as the /samples example projects. .NET Standard projects remain at version 2.0.
```
Comment (joperezr) [inline README.md:15]:
```suggestion
Whilst most of the device binding projects target .NET Core 2.1 for backward compatibility, the sample projects target the latest stable .NET Core. This applies to the sample projects with each device as well as the example projects on the `/samples` directory.
```

**Comment by djaus2** [comment]
Thanks Jose, Rich and Laurent for comments. Jose, the version differences are not material and just a question of timelines. I accept all of your suggestions.
So can this get a rap up now?

**Comment by richlander** [comment]
Looks like we can merge your changes after you address the comments from @joperezr and resolve the branch conflicts. Thanks!

**Comment by joperezr** [comment]
Took the liberty to fix the issues I called out. Thanks for the contribution @djaus2

**Comment by djaus2** [comment]
Thanks, was going to do that t his morning.

David Jones MVP

________________________________
